cpp/src/arrow/compute/kernels/aggregate_test.cc

```diff
@@ -726,6 +726,109 @@ TYPED_TEST(TestRandomNumericMinMaxKernel, RandomArrayMinMax) {
   }
 }
 
+//
+// Any
+//
+
+class TestPrimitiveAnyKernel : public ::testing::Test {
+  using Traits = TypeTraits<BooleanType>;
+  using ArrayType = typename Traits::ArrayType;
+  using c_type = typename BooleanType::c_type;
+  using ScalarType = typename Traits::ScalarType;
```

Comment:
These typedefs probably not needed, just use the Boolean* types within

Reply:
Got rid of all these